source/sass/type.scss

```diff
@@ -41,6 +41,24 @@ a {
   line-height: 1.5;
 }
 
+@mixin yellow-underline-nav-link($active-class-name, $font-family: "Zilla Slab") {
+  font-family: $font-family;
+  color: #231f20;
+
+  @if $font-family == "Zilla Slab" {
+    letter-spacing: 0.25px;
+  }
+
+  &:hover,
+  &#{$active-class-name} {
+    color: inherit;
+    text-decoration: none;
+    border-bottom: 5px solid #ffed00;
+    position: relative;
+    top: 2.5px;
```

Comment:
All browsers round subpixels differently, so its better to just use integer values with pixel units IMO.